tests/composite_pk/tests.py

```diff
@@ -151,9 +151,7 @@ def test_in_bulk_batching(self):
         Comment.objects.all().delete()
         batching_required = connection.features.max_query_params is not None
         expected_queries = 2 if batching_required else 1
-        with unittest.mock.patch.object(
-            type(connection.features), "max_query_params", 10
-        ):
+        with unittest.mock.patch.object(connection.features, "max_query_params", 10):
```

Comment:
Since `max_query_params` is now a `@cached_property`, the patch target should be the instance, not the class.